runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -486,7 +485,20 @@ else if (target.contains("bob barter") && config.swapDecant())
 		}
 		else if (target.contains("zahur"))
 		{
-			swap(config.swapZahur().getOption().toLowerCase(), option, target, false);
+			switch (config.swapZahur())
```

Comment:
To be honest, I preferred `swap(config.swapZahur().getOption().toLowerCase(), option, target, false);`. The performance hit is in a number of nanoseconds, and will not have any real impact on client functionality. I don't know how others feel about this, so I'll let others chime in if they want to.